src/main/java/com/cburch/logisim/fpga/hdlgenerator/ToplevelHDLGeneratorFactory.java

```diff
@@ -279,45 +274,42 @@ public String GetSubDir() {
       }
     }
     if (NrOfInputPorts > 0) {
-      for (int input = 0; input < NrOfInputPorts; input++) {
-        String SName =
-            "s_"
-                + CorrectLabel.getCorrectLabel(
-                    Nets.GetInputPin(input)
-                        .GetComponent()
-                        .getAttributeSet()
-                        .getValue(StdAttr.LABEL));
-        int NrOfBits = Nets.GetInputPin(input).GetComponent().getEnd(0).getWidth().getWidth();
+      for (var input = 0; input < NrOfInputPorts; input++) {
+        String SName = "s_"
```

Comment:
variable name should start with lowercase. Please rename `SName`

Reply:
see my remark above